### PR TITLE
Added aparna2071

### PR DESCRIPTION
**Pull request by aparna2071**

### CODE REVIEW

File: public/data/aparna2071.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        "description": "Led the team in creating premium technical content for CSE department"
+     },
+     {
+        "title": "Started my Open source journey on GitHub",
+        "date": "2019",
+        "icon": "github",
+        "description": "Started using GitHub"
+     }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/aparna2071.json:86]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/aparna2071.json (hunk 1 above)
Comment (aparna2071) [reply public/data/aparna2071.json:86]: Thanks @Panquesito7 
I will make the necessary changes

**Comment by aparna2071** [comment]
@Panquesito7 done with the required change

**Review by kumarsonsoff3 — APPROVED** [review]
Nice, looks good to me! 👍  Thanks for your Contribution! 🚀

**Comment by aparna2071** [comment]
Thanks @kumarsonsoff3 for approving

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by aparna2071** [comment]
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good

Thanks @emmalearnscode

File: public/data/aparna2071.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        "title": "MERN Stack developer at Accenture",
+        "date": "August 2021-Present",
+        "icon": "dollar",
+        "description": "Started job as a Full stack web developer and got recognised as a Merit holder in MERN stack training",
+        "url": "https://www.accenture.com/in-en"
+      },
+      {
+        "title": "Graduated from GGSIPU, India with a CGPA of 9.12",
+        "date": "July 2021",
+        "icon": "Graduation Hat",
```
Comment (Panquesito7) [inline public/data/aparna2071.json:48]: I believe this should have a hyphen. 👀
```suggestion
        "icon": "graduation-hat",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/aparna2071.json (hunk 2 above)
Comment (emmalearnscode) [reply public/data/aparna2071.json:48]: Well spotted, yes this should be as @Panquesito7 suggested, otherwise it won't show up

**Comment by aparna2071** [comment]
Thanks @Panquesito7 and @emmalearnscode for your suggestions.
I have made the necessary changes.

**Pull request merged**